data/json/terrain.json

```diff
@@ -2813,6 +2822,7 @@
         "type" : "terrain",
         "id" : "t_window_reinforced_noglass",
         "name": "reinforced boarded up window",
+        "description": "A heavily fortified glass window with carefully placed planks across the open frame to block vision.  Adding a few spikes and metal plating would further increase its durability.",
```

Comment:
Minor nitpick: remove 'glass' from the description, as this terrain doesn't have glass.

Reply:
Good catch, thanks